WatchDogEclipsePlugin/WatchDog/build.properties

```diff
@@ -12,8 +12,8 @@ bin.includes = META-INF/,\
                lib/commons-validator-1.4.0.jar,\
                pom.xml,\
                plugin.xml,\
+               lib/commons-lang-2.6.jar,\
                lib/jcommon-1.0.23.jar,\
                lib/jfreechart-1.0.19-swt.jar,\
-               lib/jfreechart-1.0.19.jar,\
-               lib/commons-lang-2.6.jar
+               lib/jfreechart-1.0.19.jar

```

Comment:
Can you unstage these changes? (Or is there any specific reason for changing the order?)